.github/workflows/conformance.yml

```diff
@@ -20,8 +20,8 @@ permissions:
 
 env:
   CONFORMANCE_REPO_HASH: ee6008ef151489a5330cd886b422af8f6ed58881
-  LIBJXL_VERSION: 0.9.0
-  LIBJXL_ABI_VERSION: 0.9
+  LIBJXL_VERSION:  0.9.1
```

Comment:
extra spaces; also, do be want to quote this?

Reply:
fixed.
In head the chnage to ci.sh should already make sure we are always using quotes and get the spacing right here...